Swift/BRCrypto/BRCryptoAccount.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  BRCrypto.swift
+//  BRCrypto
+//
+//  Created by Ed Gamble on 3/27/19.
+//  Copyright © 2018 Breadwallet AG. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//  See the CONTRIBUTORS file at the project root for a list of contributors.
+//
+import Foundation // Data
+import BRCryptoC
+
+///
+///
+///
+public final class Account {
+    let core: BRCryptoAccount
+
+    // A 'globally unique' ID String for account.  For BlockchainDB this will be the 'walletId'
+    let uids: String
+
+    public var timestamp: UInt64 {
```

Comment:
Can this be rolled into the create functions?

Reply:
That is CORE-316.